lib/ansible/modules/files/find.py

```diff
@@ -46,7 +46,17 @@
         version_added: "2.5"
     contains:
         description:
-            - One or more regex patterns which should be matched against the file content.
+            - match a regex pattern from the beginning of a line, in a file.
+            - if contains_anywhere is set to yes, match a regex pattern against line in a file.
+    contains_anywhere:
+        description:
+            - If no, match the contains regex pattern from the start of a line.
+            - If yes, match the contains regex pattern against a line.
+                - Please use '^' in 'contains' to search for match from the start of a line
```

Comment:
Wrap `no`, `yes`, `^`, and `contains` in `C()` so they are code formatted in the HTML docs.

Reply:
this should not be a new option, the initial approach to change to 'search' was correct, this might be 'stableinterface' but the current behaviour is a bug.